php-frontend/src/main/java/org/sonar/php/symbols/ClassSymbolIndex.java

```diff
@@ -88,6 +90,11 @@ public QualifiedName qualifiedName() {
       return Optional.ofNullable(superClass);
     }
 
+    @Override
+    public List<ClassSymbol> implementedInterfaces() {
+      return data.implementedInterfaces().stream().map(i -> get(i)).collect(Collectors.toList());
```

Comment:
Very minor but maybe we can also cache this in a property just as with the `superClass()`.

Reply:
I prefer not to do it for the moment to avoid a premature optimization.
I suppose that this method will be called much less often than `superClass()` because of the way I implemented `isOrSubClassOf` and `isSubTypeOf`.